x-pack/plugins/security/server/saved_objects/secure_saved_objects_client_wrapper.ts

```diff
@@ -617,32 +614,43 @@ export class SecureSavedObjectsClientWrapper implements SavedObjectsClientContra
     return uniq(objects.map((o) => o.type));
   }
 
-  private async getNamespacesPrivilegeMap(namespaces: string[]) {
+  private async getNamespacesPrivilegeMap(
+    namespaces: string[],
+    previouslypreviouslyAuthorizedSpaceIds: string[]
```

Comment:
typo?

```suggestion
    previouslyAuthorizedSpaceIds: string[]
```

Reply:
Yeah it was a bad "Change All Occurrences" 😝 can't believe I missed this...